.fixtures.yml

```diff
@@ -2,15 +2,19 @@ fixtures:
   forge_modules:
     stdlib: 'puppetlabs-stdlib'
     apt: 'puppetlabs-apt'
+    puppet_conf: 'puppetlabs-puppet_conf'
     archive: "puppet-archive"
     translate: "puppetlabs-translate"
     augeasproviders_sysctl: "herculesteam-augeasproviders_sysctl"
     augeasproviders_core: "herculesteam-augeasproviders_core"
     kmod: "camptocamp-kmod"
+    helm: 'puppetlabs-helm'
+    rook: 'puppetlabs-rook'
```

Comment:
you don't need to add rook in the forge_modules if you have it set in the repositories